modules/pixi/PixiLayerCustomData.js

```diff
@@ -206,7 +253,9 @@ export class PixiLayerCustomData extends AbstractLayer {
         if (feature.v !== version) {
           feature.v = version;
           feature.geometry.setCoords(coords);
-          feature.label = l10n.displayName(d.properties);
+          if (d.properties){
+            feature.label = l1n.displayName(d.properties);
```

Comment:
@bhousel - I ran into stack traces while rendering this stuff, so I made the labelling contingent on whether properties existed (there are no labels supplied for these arbitrary polys).

Reply:
Ok we can check.  I don't think wkt supports properties, but after it gets converted to GeoJSON it should have them.  I have seen some GeoJSON's in the wild where they are omitted, but you can always do something like `d.properties || {}` 

(also make sure it's `l10n` and not `l1n`)